### DISCUSSION TITLE
[question] Possible to permanently add to "Quick Swither" to the right or left panel?

### DISCUSSION THREAD

**Discussion by craziedde** [tadashi-aikawa/obsidian-another-quick-switcher#266] [General]
Hi,

Feature request in the screenshot below:
![image](https://github.com/user-attachments/assets/536caa04-18cc-4c58-823f-315b2e8eba31)

@craziedde

**Comment by tadashi-aikawa** [comment]
Hi, @craziedde :)
That is not possible because it falls outside the intended role of the Quick Switcher UI.
